pymc_experimental/statespace/filters/distributions.py

```diff
@@ -0,0 +1,78 @@
+import pymc as pm
+import pytensor
+from pymc.distributions.distribution import Distribution, SymbolicRandomVariable
+from pymc.distributions.shape_utils import get_support_shape_1d
+from pymc.pytensorf import collect_default_updates
+from pytensor.graph.basic import Node
+
+
+class LinearGaussianStateSpaceRV(SymbolicRandomVariable):
+    default_output = 1
+    _print_name = ("LinearGuassianStateSpace", "\\operatorname{LinearGuassianStateSpace}")
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Forgot to remove?

Reply:
I thought you have to do that with python subclasses

Reply:
Only if you want to do something custom